TheMovieDB/TheMovieDB/Controller/DetailViewController.swift

```diff
@@ -0,0 +1,124 @@
+        static let cornerRadius = CGFloat(5)
+    }
+    let deviceIdiom = UIScreen.main.traitCollection.userInterfaceIdiom
+    
+    init(movie: Movie) {
+        self.movie = movie
+        super.init(nibName: nil, bundle: nil)
+    }
+    
+    required init?(coder: NSCoder) {
+        return nil
+    }
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        setUpView()
+        // Do any additional setup after loading the view.
+    }
+    
+    func setUpView() {
+        loadData()
+        view.addSubview(movieImage)
+        view.addSubview(movieTitle)
+        view.addSubview(movieOverview)
+        setImageLayout()
+        setTitleLayout()
+        setOverviewLayout()
+        view.backgroundColor = Constants.backgroundColor
+    }
+    
+    func loadData() {
+        movieTitle.text = movie.title
+        movieOverview.text = movie.overview
+        if let url = movie.backdropPath {
+            movieImage.loadImage(url: url)
+        } else {
+            movieImage.image = nil
+        }
+    }
+    
+    func setImageConstraints() {
+        movieImage.translatesAutoresizingMaskIntoConstraints = false
+        let margins = view.layoutMarginsGuide
+        NSLayoutConstraint.activate([
+            movieImage.topAnchor.constraint(equalTo: margins.topAnchor, constant: Constants.margin),
+            movieImage.leadingAnchor.constraint(equalTo: margins.leadingAnchor),
+            movieImage.trailingAnchor.constraint(equalTo: margins.trailingAnchor),
+        ])
+        movieImage.heightAnchor.constraint(equalTo: view.heightAnchor, multiplier: 1/3).isActive = true
+    }
+    
+    func setTitleConstraints() {
+        movieTitle.translatesAutoresizingMaskIntoConstraints = false
+        let margins = view.layoutMarginsGuide
+        NSLayoutConstraint.activate([
+            movieTitle.topAnchor.constraint(equalTo: movieImage.bottomAnchor, constant: Constants.margin),
+            movieTitle.leadingAnchor.constraint(equalTo: margins.leadingAnchor),
+            movieTitle.trailingAnchor.constraint(equalTo: margins.trailingAnchor),
+        ])
+        
+    }
+    
+    func setOverviewConstraints() {
+        movieOverview.translatesAutoresizingMaskIntoConstraints = false
+        let margins = view.layoutMarginsGuide
+        NSLayoutConstraint.activate([
+            movieOverview.topAnchor.constraint(equalTo: movieTitle.bottomAnchor, constant: Constants.margin),
+            movieOverview.leadingAnchor.constraint(equalTo: margins.leadingAnchor),
+            movieOverview.trailingAnchor.constraint(equalTo: margins.trailingAnchor)
+        ])
+    }
+    
+    func setOverviewLayout() {
+        movieOverview.textColor = .white
+        movieOverview.numberOfLines = 0
+        movieOverview.textAlignment = .center
+        movieOverview.adjustsFontSizeToFitWidth = true
+        if deviceIdiom == .pad {
+            movieOverview.font = UIFont(name: "Copperplate", size: 25)
```

Comment:
For later you could create a method to create the copperplate font and receive the size of the font as a parameter